src/main/migrations/changesets/job_avoidance.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="chrisl" id="job_avoidance">
+        <comment>Adding some tracking columns for determining eligibility for Job Avoidance.</comment>
+        <addColumn tableName="EXECUTION">
+            <column name="ALLOWS_AVOIDANCE" type="BOOLEAN" defaultValueBoolean="true" >
+                <constraints nullable="false" />
+            </column>
+            <column name="DOCKER_IMAGE_HASH" type="VARCHAR(100)">
```

Comment:
My white whale at the moment is the Docker Image ID, not the even _more_ elusive image hash.


Btw, google results are very confusing, as specifications and implementations continue to change, while the old articles stay the same.

There are some folks [exclaiming](https://medium.com/@quayio/your-docker-image-ids-are-secrets-and-its-time-you-treated-them-that-way-f55e9f14c1a4) not to make Image ID's public, while I still haven't confirmed that gcr.io will even give out image hashes using the Docker Registry API v2, yet.

```bash
gm103-226:gcloud kshakir$ gcloud docker pull busybox:latest
latest: Pulling from library/busybox
Digest: sha256:87fcdf79b696560b61905297f3be7759e01130a4befdfe2cc9ece9234bbbab6f
Status: Image is up to date for busybox:latest

gm103-226:gcloud kshakir$ gcloud docker --server=beta.gcr.io pull beta.gcr.io/broad-dsde-dev/ubuntu
WARNING: Authenticating to a non-default server: beta.gcr.io.
Using default tag: latest
Pulling repository beta.gcr.io/broad-dsde-dev/ubuntu
91e54dfb1179: Download complete 
d3a1f33e8a5a: Download complete 
c22013c84729: Download complete 
d74508fb6632: Download complete 
Status: Image is up to date for beta.gcr.io/broad-dsde-dev/ubuntu:latest
beta.gcr.io/broad-dsde-dev/ubuntu: this image was pulled from a legacy registry.  Important: This registry version will not be supported in future versions of docker.
```